app/Platform/Actions/BuildGameShowPagePropsAction.php

```diff
@@ -725,13 +726,67 @@ private function buildSetRequestData(Game $backingGame, ?User $user): ?GameSetRe
     /**
      * @return Collection<int, LeaderboardData>
      */
-    private function buildLeaderboards(Game $game, ?User $user = null, ?int $limit = null): Collection
+    private function buildFeaturedLeaderboards(Game $game, ?User $user = null, ?int $limit = null): Collection
+    {
+        return $this->buildLeaderboards($game, $user, $limit, activeOnly: true, showUnofficial: false);
+    }
```

Comment:
Given `buildFeaturedLeaderboards()` is only one line, it seems like an unnecessary abstraction around `buildLeaderboards()`.